engines/enginetest/testutils.go

```diff
@@ -27,6 +30,12 @@ func evalNilOrPanic(f func() error, a ...interface{}) {
 	nilOrpanic(f(), a...)
 }
 
+func assert(condition bool, a ...interface{}) {
+	if !condition {
+		fmtPanic(a...)
+	}
+}
+
```

Comment:
Whatever we consider best practices for contributing code to the worker should be documented in a conventions file somewhere for new contributors.